ballerina-command/src/main/java/org/ballerinalang/command/util/OSUtils.java

```diff
@@ -67,6 +67,21 @@ public static String getExecutableFileName() {
         return null;
     }
 
+    /**
+     * Provide file name of tool update install script for current operating system.
+     * @return name of the file
+     */
+    public static String getInstallScriptFileName() {
+        if (OSUtils.isWindows()) {
+            return "install.bat";
+        } else if (OSUtils.isUnix() || OSUtils.isSolaris()) {
+            return "install";
+        } else if (OSUtils.isMac()) {
```

Comment:
this else if can be merged with above else if ?

Reply:
Implemented the same way as current getExecutableFileName method.